internal/pkg/upgrade/upgrade.go

```diff
@@ -47,6 +43,13 @@ func Upgrade(continueDirectly bool) error {
 
 	log.Debugf("Dtm upgrade: dtm file name is : %v", dtmFileName)
 
+	execDir, err := os.Executable()
```

Comment:
Why not just use this func?😂

<img width="913" alt="image" src="https://user-images.githubusercontent.com/36830265/197958122-0534b013-df69-4bab-bbbc-0171a712dfbd.png">

Reply:
I just fixed this error